hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/JsonKafkaSource.java

```diff
@@ -57,6 +57,8 @@
  */
 public class JsonKafkaSource extends KafkaSource<String> {
 
+  private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
Just being curious, do we have to use static object mapper here or it's fine to wrap it in the iterator as well?

Reply:
Good point, I think we should revert this change, because anyway the `ObjectMapper` is serde from Spark driver to executor if using the static object, so there's not much gain.